tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/RunConfigurationSelector.java

```diff
@@ -59,25 +73,245 @@ public PitRunConfiguration getConfiguration(String configName) {
+        Iterator<String> it = config.getProjects().iterator();
+        StringBuilder sb = new StringBuilder();
+        while (it.hasNext()) {
+            sb.append(it.next());
+            if (it.hasNext()) {
+                sb.append(',');
+            }
+        }
+        return sb.toString();
+    }
+
+    public void setMutatorGroup(String configurationName, Mutators mutatorGroup) {
+        activateMutatorsTab(configurationName);
+        bot.radio(mutatorGroup.getDescriptor()).click();
+        closeConfigurationShell();
+    }
+
+    public void setAdditionalCustomMutator(String configurationName, Mutators mutator) {
+        activateMutatorsTab(configurationName);
+        SWTBotRadio radioButton = bot.radio(PitMutatorsTab.CUSTOM_MUTATOR_RADIO_TEXT);
+        if (!radioButton.isSelected()) {
+            radioButton.click();
+        }
+        SWTBotTable table = bot.table();
+        setMutator(table, mutator);
+        closeConfigurationShell();
+    }
+
+    public void setOneCustomMutator(String configurationName, Mutators mutator) {
+        activateMutatorsTab(configurationName);
+        bot.radio(PitMutatorsTab.CUSTOM_MUTATOR_RADIO_TEXT).click();
+        uncheckAllMutators();
+        SWTBotTable table = bot.table();
+        setMutator(table, mutator);
+        closeConfigurationShell();
+    }
+
+    public void checkAllMutators(String configurationName) {
+        activateMutatorsTab(configurationName);
+        bot.radio(PitMutatorsTab.CUSTOM_MUTATOR_RADIO_TEXT).click();
+        SWTBotTable table = bot.table();
+        Display.getDefault().syncExec(() -> {
+            final int itemCount = table.widget.getItems().length;
+            for (int i = 0; i < itemCount; i++) {
+                table.getTableItem(i).check();
+            }
+        });
+        closeConfigurationShell();
+    }
+
+    /**
+     * Assumes the Mutator tab is active and custom mutator mode is selected.
+     */
+    private void uncheckAllMutators() {
+        SWTBotTable table = bot.table();
+        Display.getDefault().syncExec(() -> {
+            for (TableItem item : table.widget.getItems()) {
+                item.setChecked(false);
+            }
+        });
+        // don't close, because you cannot apply empty mutators
+    }
+
+    /**
+     * Assumes the Mutator tab is active and custom mutator mode is selected.
+     * @param table   where to check the mutator
+     * @param mutator which is set to active
+     */
+    private void setMutator(SWTBotTable table, Mutators mutator) {
+        table.getTableItem(mutator.getDescriptor()).check();
+    }
+
+    /**
+     * Closes the configuration shell and applies, if possible.
+     */
+    private void closeConfigurationShell() {
+        SWTBotShell shell = bot.shell(RUN_CONFIGURATIONS);
+        SWTBotButton apply = shell.bot().button("Apply");
+        if (apply.isEnabled()) {
+            apply.click();
+        }
+        shell.close();
```

Comment:
As I said in the past, it's best practice to wait for the shell to close before going on, to avoid flaky tests